packages/rlc-common/src/metadata/packageJson/buildAzureMonorepoPackage.ts

```diff
@@ -193,6 +193,7 @@ function getAzureMonorepoScripts(config: AzureMonorepoInfoConfig) {
     "build:samples": "dev-tool samples publish --force",
     "check-format":
       'dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore "src/**/*.{ts,cts,mts}" "test/**/*.{ts,cts,mts}" "*.{js,cjs,mjs,json}"',
+    "clean": "rimraf --glob dist dist-browser dist-esm test-dist temp types *.tgz *.log",
```

Comment:
what's the build script for non-azure ? wonder if the clean step also necessary for non azure? if yes, we probably missing something?

Reply:
https://github.com/Azure/autorest.typescript/blob/main/packages/rlc-common/src/metadata/packageJson/packageCommon.ts#L132 in non-azure it is already there.

Reply:
is it a little weird that we don't leverage the packageCommon.ts in azure monorepo?

Reply:
yeah i guess because most of test relevant build and steps are a little different.